<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>273</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_num1">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>195</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Число 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_num2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>195</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Число 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_result">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Результат</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_sum">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>195</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Сложение</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton_diff">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>195</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Вычитание</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_mult">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>195</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Умножение</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioButton_div">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>195</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Деление</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(34, 214, 21);
color: #ffffff</string>
        </property>
        <property name="text">
         <string>Выполнить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>exec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exec()</slot>
 </slots>
</ui>
